src/Amadeus/Client/RequestOptions/Service/StandaloneCatalogue/Segment.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
If this is a copy of an existing file, use the existing file instead of copying it.

Reply:
done, I deleted this file.